tests/op/test_model_function.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2024 NVIDIA CORPORATION
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from unittest.mock import patch
+
+cp = pytest.importorskip("cupy")
+cudf = pytest.importorskip("cudf")
+dask_cudf = pytest.importorskip("dask_cudf")
+dd = pytest.importorskip("dask.dataframe")
+pd = pytest.importorskip("pandas")
+transformers = pytest.importorskip("transformers")
+torch = pytest.importorskip("torch")
+
+import crossfit as cf  # noqa: E402
+
+
+cf_loader = pytest.importorskip("crossfit.backend.torch.loader")
+
+
+@pytest.mark.parametrize("trust_remote_code", ["y"])
+def test_model_output_int(trust_remote_code, model_name="ai4bharat/indictrans2-en-indic-1B"):
```

Comment:
As we are not really using the `ai4bharat/indictrans2-en-indic-1B` here , can we just switch this to a diffrent model like `microsoft/deberta-v3-base`.

Reply:
ok let me test it